dev/io.openliberty.io.smallrye.openapi41.core/bnd.bnd

```diff
@@ -0,0 +1,46 @@
+#*******************************************************************************
+# Copyright (c) 2024, 2025 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/rejar.props
+
+bVersion=1.0
+
+Bundle-SymbolicName: io.openliberty.io.smallrye.openapi41.core; singleton:=true
+
+WS-TraceGroup: MPOPENAPI
+
+javac.source: 11
+javac.target: 11
+
+Require-Capability: osgi.ee; filter:="(&(osgi.ee=JavaSE)(version=11))"
+
+Include-Resource: \
+    resources
+
+Import-Package: \
+    !io.xlate.yamljson.*,\
+    !jakarta.json.*,\
+    *
+
+Export-Package: \
+    io.smallrye.openapi.api.*;version=4.0,\
+    io.smallrye.openapi.internal.models.*;version=4.0,\
+    io.smallrye.openapi.internal.support.*;version=4.0,\
+    io.smallrye.openapi.runtime.*;version=4.0,\
+    io.smallrye.openapi.spi.*;version=4.0
```

Comment:
I assume these will change to 4.1 once you switch to the actual 4.1 version of small rye.  Obviously you could do it now before that happens.

Reply:
Yes. I have this pending on another branch already so I won't do it here.